code/mod_table/mod_table.cpp

```diff
@@ -55,6 +55,7 @@ std::tuple<ubyte, ubyte, ubyte> Arc_color_damage_s1;
 std::tuple<ubyte, ubyte, ubyte> Arc_color_emp_p1;
 std::tuple<ubyte, ubyte, ubyte> Arc_color_emp_p2;
 std::tuple<ubyte, ubyte, ubyte> Arc_color_emp_s1;
+bool Use_engine_wash_intensity;
```

Comment:
This is missing a default value in `mod_table_reset`.

Reply:
Thanks, added.